src/style/design-tokens/generate-css-variables.util.js

```diff
@@ -7,7 +8,10 @@
  *
  */
 
+const baseThemeKeys = Object.keys(baseTheme);
+
 export default (theme) =>
   Object.entries(theme)
+    .filter(([key]) => !baseThemeKeys.includes(key))
```

Comment:
When the `sage` theme is used this still results in additional properties that will not be used.

![image](https://user-images.githubusercontent.com/2328042/140783341-4efde675-d0e3-4a7d-a4fa-ba16e0bc140a.png)


From our previous discussion I think we should introduce a `compatibility` attribute into the existing themes and only properties within that object should be converted to css-variables.

e.g.

```diff
export const mergeWithBase = (configureTheme) => {
  const themeToMergeWithBase = configureTheme(palette);
  const mergedThemes = {
    ...mergeDeep(baseTheme, themeToMergeWithBase),
    palette,
  };

  return {
    ...mergedThemes,
-   ...compatibility(mergedThemes),
+   compatibility: compatibility(mergedThemes),
    name: mergedThemes.name,
  };
};
```

Reply:
About typography tokens - those will be exported in Design Tokens Library in right format. We are working on it.